managers/backup-manager/BackupService.js

```diff
@@ -285,14 +285,21 @@ class BackupService extends BaseDirectorService {
                 logs: logs,
                 snapshotId: lastOperation.snapshotId
               })
+              .return({
+                state: lastOperation.state,
+                logs: logs,
+                snapshotId: lastOperation.snapshotId,
+                finished_at: new Date(Date.now())
```

Comment:
If we are setting the date here, will the finished at value in backupstore and in apiserver would be different isn't it?

Reply:
yes . but since out least count is in minutes , this should be acceptable ? 
Or we can pass down the date created here down the function chain .

Reply:
fine then.